core-libs/setup/ssr/providers/ssr-providers.ts

```diff
@@ -26,9 +33,21 @@ export function provideServer(options?: ServerOptions): StaticProvider[] {
       provide: SERVER_REQUEST_URL,
       useFactory: serverRequestUrlFactory(options),
     },
+    {
+      provide: ErrorHandler,
+      useFactory: ssrErrorHandlerFactory,
+    },
```

Comment:
[MEDIUM]

When we move CxErrorHandler to the `@spartacus/core` lib, let's move also this provider to the core ErrorModule

PS. `/error` vs. `/error-handling` .  `ErrorModule` vs `ErrorHandlingModule` ?

Reply:
I prefer the second option as it provides more context, it's in line with the naming convention known from Angular.